### ISSUE TITLE
[BUG] fmdns import zone file about tabs as delimeter problem

### ISSUE THREAD

**Issue by marksmansj** [WillyXJ/facileManager#488]
Please prefix your issue title with one of the following: [BUG], [ISSUE], [FEATURE REQUEST], [MODULE REQUEST], [OTHER].

Replace everything between stars with current version of your facileManager and module installations:  
fM Version : **3.5.7**
{Module Name} Version : **3.4.1**

**In raising this issue, I confirm the following (please check boxes, eg [X]):**

- [X] I have read and understood the [contributors guide](https://github.com/WillyXJ/facileManager/blob/master/.github/CONTRIBUTING.md).
- [X] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [X] I have checked that the issue I'm posting isn't already reported.
- [X] I have checked that the issue I'm posting isn't already solved and no duplicates exist in [closed issues](https://github.com/WillyXJ/facileManager/issues?q=is%3Aissue+is%3Aclosed) and [opened issues](https://github.com/WillyXJ/facileManager/issues)
- [X] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

---

**(BUG | ISSUE) Expected Behavior:**

**(BUG | ISSUE) Actual Behavior:**

**(BUG | ISSUE) Steps to reproduce:**

{ when I imported the bind dump file into the fmdns. I found the records were not imported into fmdns. I checked the bind dump file and found there are tabs instead of spaces between the TTL and IN. then I replaced the tabs to spaces and imported the dump file again and the the records were imported successfully.  }

**Comment by marksmansj** [comment]
@WillyXJ  Please test this problem if you are free.  I really appreciate it .

**Comment by WillyXJ** [comment]
I'm not able to reproduce this issue with test data.  In addition, there is code in the import wizard to replace the tabs.

```
/** Change tabs into spaces */
$clean_contents = str_replace("\t", ' ', $clean_contents);
```

Any chance of providing one line that does not import from your import file?

**Comment by marksmansj** [comment]
[tjapi_named_dump.zip](https://github.com/WillyXJ/facileManager/files/4699529/tjapi_named_dump.zip)
@WillyXJ  Please check attach file , there is a tab between 5 and IN , in my case . if I replace the tab into a space then I can import record successfully . thank you so much

**Comment by WillyXJ** [comment]
The records in that file imported fine on my dev system. What version of PHP is your fM server using?

**Comment by marksmansj** [comment]
> The records in that file imported fine on my dev system. What version of PHP is your fM server using?

fM Version : 3.5.7
{fmDNS} Version : 3.4.1

cat /etc/redhat-release 
CentOS Linux release 7.7.1908 (Core)

php -v
PHP 7.3.17 (cli) (built: Apr 14 2020 08:29:22) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.17, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.17, Copyright (c) 1999-2018, by Zend Technologies

**Comment by WillyXJ** [comment]
There have been lots of reports of issues with php 7.3 and above.

**Comment by marksmansj** [comment]
> There have been lots of reports of issues with php 7.3 and above.

what version do you recommend ？

**Comment by WillyXJ** [comment]
Anything below 7.3 seems to work fine.

**Comment by marksmansj** [comment]
> Anything below 7.3 seems to work fine.

thank you so much.

**Comment by marksmansj** [comment]
Hello. I use the yum remove php* then install php72 and other packages.  I restarted the httpd and mysqld service. now I imported the attach that I sent you before. after I imported this dump file. still have only one NS record  without A record.  how odd.

php -v
PHP 7.2.31 (cli) (built: May 12 2020 10:09:06) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.31, Copyright (c) 1999-2018, by Zend Technologies

**Comment by WillyXJ** [comment]
I don't think this is a PHP 7.3 issue as I've been able to reproduce the issue using the bulk import wizard.

**Comment by marksmansj** [comment]
> I don't think this is a PHP 7.3 issue as I've been able to reproduce the issue using the bulk import wizard.

thank you so much

**Comment by WillyXJ** [comment]
This is now fixed in [fmDNS 4.0.1](http://www.facilemanager.com/modules/fmdns/) and later.
